### ISSUE TITLE
4.0.2: processTicksAndRejections error

### ISSUE THREAD

**Issue by balza** [advanced-security/maven-dependency-submission-action#71]
Hello,
I've just updated the Action to V4 in a very simple workflow.
Once submitted the snapshot i get a 500 error 

```
Error: HTTP Status 500 for request POST https://-MYREPO-/dependency-graph/snapshots
Error: Response body:
{
  "message": "An error occurred while processing your request. Please try again later.",
  "documentation_url": "https://docs.github.com/enterprise-server@3.9/rest/reference/dependency-graph#create-a-snapshot-of-dependencies-for-a-repository"
}
Error: An error occurred while processing your request. Please try again later.
Error: HttpError: An error occurred while processing your request. Please try again later.
    at /home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4/node_modules/@octokit/request/dist-node/index.js:124:1
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at L (/home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4/node_modules/@github/dependency-submission-toolkit/dist/index.js:6:1)

/home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4/node_modules/@github/dependency-submission-toolkit/dist/index.js:7
${JSON.stringify(n.response.data,void 0,2)}`)),n instanceof Error&&(a.error(n.message),n.stack&&a.error(n.stack)),new Error(`Failed to submit snapshot: ${n}`)}}
^
Error: Failed to submit snapshot: HttpError: An error occurred while processing your request. Please try again later.
    at L (/home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4/node_modules/@github/dependency-submission-toolkit/dist/index.js:7:1)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

thanks

**Comment by pctapia** [comment]
Any feedback on this? We also face the same issue for some projects when using version 4. 
Action is as follows:
      - name: Submit Dependency Snapshot
        uses: advanced-security/maven-dependency-submission-action@v4.0.2

Exception

${JSON.stringify(n.response.data,void 0,2)}`)),n instanceof Error&&(a.error(n.message),n.stack&&a.error(n.stack)),new Error(`Failed to submit snapshot: ${n}`)}}
^
Error: Failed to submit snapshot: HttpError: An error occurred while processing your request. Please try again later.
    at L (/home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4.0.2/node_modules/@github/dependency-submission-toolkit/dist/index.js:7:1)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
Error: HTTP Status 500 for request POST https://github.dom.appd.com/api/v3/repos/projects/app/dependency-graph/snapshots
Error: Response body:
{
  "message": "An error occurred while processing your request. Please try again later.",
  "documentation_url": "https://docs.github.com/enterprise-server@3.12/rest/dependency-graph/dependency-submission#create-a-snapshot-of-dependencies-for-a-repository"
}
Error: An error occurred while processing your request. Please try again later.
Error: HttpError: An error occurred while processing your request. Please try again later.
    at /home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4.0.2/node_modules/@octokit/request/dist-node/index.js:124:1
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at L (/home/runner/_work/_actions/advanced-security/maven-dependency-submission-action/v4.0.2/node_modules/@github/dependency-submission-toolkit/dist/index.js:6:1)

**Comment by balza** [comment]
I don't undestand this error:
I have a repo that generate the error I reported, I forked this repository to a new repo so the content are exactly the same, I run the action on the forked repository and the action is green.

**Comment by peter-murray** [comment]
Is the version of GHES the same as when reported? i.e. 3.9?

**Comment by peter-murray** [comment]
Is it possible to check the logs on the GitHub Enterprise Server, as a 500 error will be coming from the server side and the logs there should have some additional context as to what is breaking there.

**Comment by balza** [comment]
thanks @peter-murray, the version is _GitHub Enterprise Server 3.9.6_
I've to open a ticket to internal support to have more information on logs

**Comment by balza** [comment]
The internal support opened a ticket to github and at the moment, they haven't given me further details.

**Comment by peter-murray** [comment]
Can you provide the ticket number? I have just tested this under GitHub Enterprise Server 3.11.4 and the latest version of the action, 4.0.3 and it did not produce any errors.

**Comment by balza** [comment]
I don't have visibility on this information, my internal support said
"GitHub Support shared that they have raised an internal ticket with the dependency-graph team to look into this issue."

**Comment by balza** [comment]
The issue was resolved upgrading github to version 3.11.x
